pandas_transform/pandas_transform.md

```diff
@@ -224,93 +224,81 @@ This data is saved as a [csv file hosted on GitHub](https://raw.githubuserconten
 
 </div>
 
-### Loading data
-@sage
-<div class="python_data_init">
-<lia-keep>
-<script type="text/x-sage">
-import pandas as pd
-</script>
-</lia-keep>
-</div>
+### <span class="fa-regular fa-table"></span> Loading data
```

Comment:
```suggestion
### <span class="fa-regular fa-table"></span> The data
```
I'm suggesting we use standard headings for data sections, as we do for lesson prep, overview, additional resources, and feedback.